controllers/utils/utils.go

```diff
@@ -186,8 +186,8 @@ func GetIstioControlPlaneName(ctx context.Context, cli client.Client) (istioCont
 		log.V(1).Info("Trying to read Istio Control Plane name and namespace from DSCI")
 		objectList, err := getDSCIObject(ctx, cli)
 		if err != nil {
-			log.V(0).Error(err, "Failed to fetch the DSCI object, using default values")
-			return constants.IstioControlPlaneName, constants.IstioNamespace
```

Comment:
For the time being, it may be better to invoke `panic`, rather than returning the error string in the namespace return value.

Reply:
got it, thanks for the input!!

Reply:
actually, it is logging an error and returning the default values, keeping backward capabilities.